dbpedia_ranking/challenges/default.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "dbpedia-ranking",
+  "description": "Performs relevance benchmarking using the mets/dbpedia dataset",
+  "default": true,
+  "schedule": [
+    {
+      "operation": "delete-index"
+    },
+    {
+      "operation": {
+        "include-in-reporting": false,
+        "operation-type": "create-index"
+      }
+    },
+    {
+      "name": "check-cluster-health",
+      "operation": {
+        "include-in-reporting": false,
+        "operation-type": "cluster-health",
+        "index": "dbpedia-ranking-collection",
+        "request-params": {
+          "wait_for_status": "{{cluster_health | default('yellow')}}",
+          "wait_for_no_relocating_shards": "true"
+        },
+        "retry-until-success": true
+      }
+    },
+    {
+      "operation": {
+        "operation-type": "bulk",
+        "bulk-size": {{bulk_size | default(500)}},
+        "ingest-percentage": {{ingest_percentage | default(100)}}
+      },
+      "clients": {{bulk_indexing_clients | default(8)}}
+    },
+    {
+      "name": "refresh-after-index",
+      "operation": {
+        "operation-type": "refresh",
+        "request-timeout": 1000,
+        "include-in-reporting": true
+      }
+    },
+    {
+      "name": "wait-until-merges-finish-after-index",
+      "operation": {
+        "operation-type": "index-stats",
+        "index": "_all",
+        "condition": {
+          "path": "_all.total.merges.current",
+          "expected-value": 0
+        },
+        "retry-until-success": true,
+        "include-in-reporting": false
+      }
+    },
+    {
+      "name": "default-text-search-relevance",
+      "operation": "default-text-search-relevance"
+    },
+    {
+      "name": "english-analyzed-text-search-relevance",
+      "operation": "english-analyzed-text-search-relevance"
+    }
```

Comment:
Let's also report the search throughput to compare the cost of the strategies?

Reply:
Any idea why I'm only seeing 100 pctile latency when running this, instead of a breakdown?